src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+                    updateRemove(delete(description));
+                    break;
+                default:
+                    throw new DukeException("OOPS!!! I'm sorry, but I don't know what that means :-(");
+            }
+
+            input = sc.nextLine();
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+        byeFlag = true;
+    }
+
+    public static String getCommand(String input) {
+        String command;
+
+        if (input.contains(" ")) {
+            int index = input.indexOf(" ");
+            command = input.substring(0, index);
+        } else {
+            command = input;
+        }
+
+        return command;
+    }
+
+    public static String getDescription(String input, String command) throws DukeException {
+        String action = "";
+
+        if (input.contains(" ") && input.contains("/")) {
+            int index = input.indexOf(" ") + 1;
+            String removeCommand = input.substring(index);
+            String[] split = removeCommand.split("/", 2);
+            action = split[0];
+        } else if (input.contains(" ")) {
+            String[] split = input.split(" ", 2);
+            action = split[1];
+        }
+
+        if (action.equals("")) {
+            throw new DukeException("OOPS!!! The description of a " + command + " cannot be empty.");
+        }
+
+        return action;
+    }
+
+    public static DateTime getDateAndTime(String input, String command) throws DukeException {
+        String detail = "";
+        DateTime dateTime;
+
+        if (input.contains(" ") && input.contains("/")) {
+            int index = input.indexOf(" ");
+            String removeCommand = input.substring(index);
+            String[] split = removeCommand.split("/", 2);
+            index = split[1].indexOf(" ") + 1;
+            detail = split[1].substring(index);
+        }
+
+        if (detail.equals("")) {
+            throw new DukeException("OOPS!!! The time and date of a " + command + " cannot be empty.");
+        } else {
+            dateTime = new DateTime(detail);
+            dateTime.setDateAndTime();
+        }
+
+        return dateTime;
+    }
+
+    public static void updateAdd(Task task) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println("    " + task);
+        if (taskList.size() > 1) {
+            System.out.println("Now you have " + taskList.size() + " tasks in the list.");
+        } else {
+            System.out.println("Now you have " + taskList.size() + " task in the list.");
+        }
+    }
+
+    public static void updateRemove(Task task) {
+        System.out.println("Noted. I've removed this task:");
+        System.out.println("    " + task);
+
+        if (taskList.size() > 1) {
+            System.out.println("Now you have " + taskList.size() + " tasks in the list.");
+        } else {
+            System.out.println("Now you have " + taskList.size() + " task in the list.");
```

Comment:
Instead of writing the same sentence twice, you can just do something like that to make the code neater:

```
String task = "task";
if (taskList.size() > 1) {
    task = "tasks";
}
System.out.println(String.format("Now you have %d %s in the list", taskList.size(), task);
```